src/main/java/com/google/devtools/common/options/OptionDocumentationCategory.java

```diff
@@ -112,4 +112,9 @@
    * specific inputs.
    */
   GENERIC_INPUTS,
+
+  /**
+   * This option lets a user to configure the remote caching and/or the remote execution for a build.
```

Comment:
nit: Add category of options to configure Bazel's remote caching and execution capabilities